packages/py-api/py_api/controllers/uploader_controller.py

```diff
@@ -69,3 +69,23 @@ def dump_objects(cls) -> Dict[str, Any]:
             )
 
         return {'objects': objects}
+
+    @classmethod
+    def delete_object_by_filename(cls, filename: str) -> Dict[str, Any]:
+        try:
+            if not filename:
+                return {"error": "Invalid filename"}
+
+            object_key = filename
+
+            s3 = boto3.client(
```

Comment:
I think this should be extracted as a class variable and not initialize it on each call. Keep the connection persistent.